src/modules/Invoice/pdf_template/default-pdf.twig

```diff
@@ -0,0 +1,78 @@
+{% if html_buyer_lines >= html_seller_lines %}
+	{% set top = 325 + (25 * html_buyer_lines) %}
+{% else %}
+	{% set top = 325 + (25 * html_seller_lines) %}
+{% endif %}
+
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta http-equiv='Content-Type' content='text/html; charset=utf-8'/>
+		<title>{{ invoice.serie_nr }}</title>
+		<style>
+			{{css}}</style>
+	</head>
+	<body>
+		{{ html_logo|raw }}
+		<hr class='Rounded'>
+		<div class='InvoiceInfo'>
+			<p>Invoice number:
+				{{ invoice.serie_nr }}</p>
+			<p>Invoice date:
+				{{ invoice.created_at|fossbilling_format_time }}</p>
```

Comment:
I would rather not have a filter that implements existing functionality, but under a name that's exclusive to the PDF generation. That's imo messy and not at all intuitive. 
We already have a class that defines all of the custom twig filters: https://github.com/FOSSBilling/FOSSBilling/blob/main/src/library/Box/TwigExtensions.php

Reply:
Maybe I am blind but I do not see a format time filter

Reply:
For date formatting you can use the twig filters, including the ones in the `IntlExtension` extension as we do include that.
In this case, the [format_datetime](https://twig.symfony.com/doc/3.x/filters/format_datetime.html#format-datetime) twig filter should do the trick. Just use it directly without passing any of the options as those should already be preconfigured by the DI